src/api/Recording.api.test.js

```diff
@@ -0,0 +1,118 @@
+import crossFetch from 'cross-fetch'; // mocked __mocks__/cross-fetch.js
+import recordingApi from './Recording.api';
+import {LocalStorageMock} from '../tests/testUtils';
+import { Config } from '../../app.config' // eslint-disable-line
+
+global.localStorage = new LocalStorageMock();
+
+const recordingApiUrl = '/api/v1/recordings';
+
+describe('query() calls recordings api via fetch', () => {
+  const testToken = 'testJWT';
+  const testLimit = 100;
+  const testOffset = 0;
+  const testTagMode = 'any';
+  const testTags = '';
+  const testWhere = JSON.stringify({type: 'thermalRaw'})
+  const testOrder = null
+
+  test('query() call with where, limit, offset params', async () =>{
+    let params = {
+      where: testWhere,
+      limit: testLimit,
+      offset: testOffset
+    }
+    let queryString = Object.keys(params).map((key) => {
+      return encodeURIComponent(key) + '=' + encodeURIComponent(params[key]);
+    }).join('&');
+    let expectedUrl = `${Config.api}` + recordingApiUrl + "?" + queryString;
```

Comment:
`${Config.api}` + recordingApiUrl + "?" + queryString;
Can be simplified to: 
Config.api + recordingApiUrl + "?" + queryString;